docs/setup/minikube/getting-started-minikube.md

```diff
@@ -15,35 +15,35 @@ Follow this step-by-step guide to set up Nuclio on [Minikube](https://github.com
 
+- [Docker](https://docs.docker.com/get-docker/)
 - [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+- [Helm](https://helm.sh/docs/intro/install/)
 
 It's recommended that you use these drivers:
 
-- For **Mac OS** - [hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver)
-- For **Linux** - [virtualbox](https://www.virtualbox.org/wiki/Linux_Downloads)
+- For **Mac OS** - [Docker](https://minikube.sigs.k8s.io/docs/drivers/docker/)
+- For **Linux** - [Docker](https://minikube.sigs.k8s.io/docs/drivers/docker/) or [KVM2](https://minikube.sigs.k8s.io/docs/drivers/kvm2/)
+- For **Windows** - [Docker](https://minikube.sigs.k8s.io/docs/drivers/docker/) or [Hyper-V](https://minikube.sigs.k8s.io/docs/drivers/hyperv/)
 
 ## Prepare Minikube
 
-**Start Minikube** as you normally would. Note that the following command also enables role-based access control (RBAC) (which is disabled by default on Minikube version 0.24.1 and later) so that you can get more comfortable working with an RBAC-enabled Kubernetes cluster:
+**Start Minikube** as you normally would.
+Note that the following command also enables role-based access control (RBAC) so that you can get more comfortable working with an RBAC-enabled Kubernetes cluster:
 
-- **Mac OS**
-
-    ```sh
-    minikube start --vm-driver=hyperkit --extra-config=apiserver.authorization-mode=RBAC
-    ```
-- **Linux**
+```sh
+minikube start --kubernetes-version v1.17.9 --driver docker --extra-config=apiserver.authorization-mode=RBAC --addons ingress
+```
 
-    ```sh
-    minikube start --vm-driver=virtualbox --extra-config=apiserver.authorization-mode=RBAC
-    ```
+> **Note:** You may want to
+> 1. Change the Kubernetes version. Currently, the recommended version is 1.17.9.
+> 2. Add `--addons ingress` to your `minikube start` command to support creating function ingresses.
 
-**Set admin permissions:** bypass Minikube configuration issues by giving cluster-admin permissions to the Kubernetes services, so that services such as `kube-dns` can work in Minikube:
-> **Note:** You are encouraged to look at the [**kubedns-rbac**](https://github.com/nuclio/nuclio/blob/master/hack/minikube/resources/kubedns-rbac.yaml) file that's used in the following command, and the RBAC configuration files used in the [Nuclio installation](#install-nuclio) section, before applying the files, so that you don't get into the habit of blindly running things on your cluster (akin to running scripts off the internet as root).<br/>
-> If you don't want to elevate your Kubernetes services, run Minikube with RBAC disabled (omit `--extra-config` from `minikube start`) and skip the RBAC related commands in the Nuclio installation instructions.
+> Ensure that your function ingress appears on your hosts file (**/etc/hosts**).
+> You can do this by running this command:
+> ```sh
+> echo "$(minikube ip) my-function.info" | sudo tee -a /etc/hosts
+> ```
```

Comment:
In the current version, the "Ensure ..." text appears as a separated quoted block, after the note with notes 1 & 2, but I think this should be part of note #2. Also, I would use an unordered (bulleted) list rather than an ordered (enumerated) list for the notes, because these aren't sequential steps (although at times it could be beneficial to number notes, especially when there are a lot of them). => I suggest changing to
```markdown
> **Note:** You may want to
> - Change the Kubernetes version. Currently, the recommended version is 1.17.9.
> - Add `--addons ingress` to your `minikube start` command to support creating function ingresses.
>     Ensure that your function ingress appears on your hosts file (**/etc/hosts**).
>     You can do this by running this command:
>     ```sh
>     echo "$(minikube ip) my-function.info" | sudo tee -a /etc/hosts
>     ```
```
=> Which should result in this output:
> **Note:** You may want to
> - Change the Kubernetes version. Currently, the recommended version is 1.17.9.
> - Add `--addons ingress` to your `minikube start` command to support creating function ingresses.
>     Ensure that your function ingress appears on your hosts file (**/etc/hosts**).
>     You can do this by running this command:
>     ```sh
>     echo "$(minikube ip) my-function.info" | sudo tee -a /etc/hosts
>     ```